### PR TITLE
Add flag to configure production of DDU-based trigger synch plots [10_1_X]

### PR DESCRIPTION
**Pull request by battibass**
This PR is a backport of [PR23071](https://github.com/cms-sw/cmssw/pull/23071), intended to fix the T0 issue related to pathALCARECODtCalib reported in [this HN thread](https://hypernews.cern.ch/HyperNews/CMS/get/tier0-Ops/1862/1.html).

What it does is making the production of a few plots (hence the access to some products) configurable and disables them in the 2018 era.

### CODE REVIEW

**Comment by fabiocos** [comment]
please test

**Comment by jfernan2** [comment]
+1

**Comment by fabiocos** [comment]
+1

File: DQM/DTMonitorModule/python/dtTriggerSynchTask_cfi.py [hunk 1]
```diff
@@ -27,4 +28,8 @@
     tTrigMode = cms.string('DTTTrigSyncFromDB')
 )
 
+from Configuration.Eras.Modifier_run2_DT_2018_cff import run2_DT_2018
+run2_DT_2018.toModify(dtTriggerSynchMonitor,processDDU = cms.bool(False))
```
Comment (slava77) [inline DQM/DTMonitorModule/python/dtTriggerSynchTask_cfi.py]: this has to be simply ```processDDU = False```
type specs are not needed
https://hypernews.cern.ch/HyperNews/CMS/get/tier0-Ops/1875/1/1.html

**Pull request merged**